charts/spire/charts/spiffe-oidc-discovery-provider/README.md

```diff
@@ -26,8 +26,7 @@ A Helm chart to install the SPIFFE OIDC discovery provider.
 | config.acme.directoryUrl | string | `"https://acme-v02.api.letsencrypt.org/directory"` |  |
 | config.acme.emailAddress | string | `"letsencrypt@example.org"` |  |
 | config.acme.tosAccepted | bool | `false` |  |
-| config.domains[0] | string | `"localhost"` |  |
-| config.domains[1] | string | `"oidc-discovery.example.org"` |  |
+| config.domains | list | `["localhost"]` | Override the domains to use. (jwtIssuer is automatically added to this list). |
```

Comment:
The need for the comment here is an indication that the API is not expressive enough to hold up without the additional commentary.

I can see two paths forward on the API side.

1. Keep the "config.domains" key, and add a _helpers.tpl test to ensure that it includes the required matching jwtIssuer domain.
2. Alter the "config.domains" key to be named "config.additionalDomains" or something like that.

One is clearly more work than the other, but I haven't thought about the other aspects of each choice yet.  There are likely other options available too.  If you address this in a different way, that's fine by me; I'd like you to consider which way would service your needs the best.

Reply:
I like the idea of renaming the option to be more representative of what it does. additionalDomains makes sense to me. The rename would be easy. I'll submit that and if we want to go the other way, we can do that instead.

Reply:
Done

Reply:
Verified changed to additional domains